pom.xml

```diff
@@ -370,7 +370,7 @@
                 src/main/webapp/sass:src/main/webapp/generated
               </arguments>
             </configuration>
-            <phase>compile</phase>
+            <phase>process-resources</phase>
```

Comment:
Move `sass` compilation to `process-resources` to ensure the static resources created by this job are copied, which happens in the `compile` step